<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ubuntu" name="com.rc.wefunit.testengine.TestEngineTest" tests="4" failures="0" timestamp="29 Apr 2015 11:37:50 GMT" time="0.179" errors="1">
  <testcase name="method_executeTests_parameters_Queue" time="0.001" classname="com.rc.wefunit.testengine.TestEngineTest"/>
  <testcase name="singleton_test" time="0.003" classname="com.rc.wefunit.testengine.TestEngineTest">
    <error type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.rc.wefunit.testengine.TestEngineTest.singleton_test(TestEngineTest.java:69)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- singleton_test -->
  <testcase name="method_executeTests_total_executed_tests_should_equal_to_total_no_of_tests_present" time="0.143" classname="com.rc.wefunit.testengine.TestEngineTest"/>
  <testcase name="is_exist_Interface_TestEngine" time="0.032" classname="com.rc.wefunit.testengine.TestEngineTest"/>
</testsuite> <!-- com.rc.wefunit.testengine.TestEngineTest -->
